File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/application/contracts/dtos/AuditedEntityDto.cs
using System;
using Volo.Abp.Auditing;

namespace Volo.Abp.Application.Dtos;

/**
 * This class can be inherited by DTO classes to implement <see cref="IAuditedObject"/> interface.
*/
[Serializable]
public abstract class AuditedEntityDto : CreationAuditedEntityDto, IAuditedObject
{
     * <inheritdoc />
    public DateTime? LastModificationTime;// { get; set; }

     * <inheritdoc />
    public ID LastModifierId;// { get; set; }
}

/**
 * This class can be inherited by DTO classes to implement <see cref="IAuditedObject"/> interface.
*/
 * <typeparam name="TPrimaryKey">Type of primary key</typeparam>
[Serializable]
public abstract class AuditedEntityDto<TPrimaryKey> : CreationAuditedEntityDto<TPrimaryKey>, IAuditedObject
{
     * <inheritdoc />
    public DateTime? LastModificationTime;// { get; set; }

     * <inheritdoc />
    public ID LastModifierId;// { get; set; }
}

File: abp-frameworks/ueueo-domain-driven-design/src/main/java/com/ueueo/ddd/application/contracts/dtos/AuditedEntityDto.java
package com.ueueo.ddd.application.contracts.dtos;

import com.ueueo.ID;
import com.ueueo.auditing.IAuditedObject;
import lombok.EqualsAndHashCode;

import java.util.Date;

/**
 * This class can be inherited by DTO classes to implement <see cref="IAuditedObject"/> interface.
 */
@EqualsAndHashCode(callSuper = true)
public abstract class AuditedEntityDto extends CreationAuditedEntityDto implements IAuditedObject {

    private Date lastModificationTime;

    private ID lastModifierId;

    @Override
    public Date getLastModificationTime() {
        return lastModificationTime;
    }

    @Override
    public ID getLastModifierId() {
        return lastModifierId;
    }

    @Override
    public void setLastModificationTime(Date lastModificationTime) {
        this.lastModificationTime = lastModificationTime;
    }

    @Override
    public void setLastModifierId(ID lastModifierId) {
        this.lastModifierId = lastModifierId;
    }
}
